### PR TITLE
Files download

### PR DESCRIPTION
**Pull request by laxmikandel97**
Admin can download the files uploaded by tutors. All the tutor uploaded files are shown in the tutorInfo a page. Downloading all files at once is not done yet!

### CODE REVIEW

File: views/tutorInfo.html [hunk 1]
```diff
@@ -14,12 +14,23 @@ <h2 class="mt-3 text-center">{{ @tutor["tutor_first"] }} {{ @tutor["tutor_last"]
             <p class="mt-3"><span class="font-weight-bold mr-1">SSN:</span> {{ @ssn }}</p>
             <p class="mt-3"><span class="font-weight-bold mr-1">Bio:</span> {{ @tutor["tutor_bio"] }}</p>
             <form action="#" method="get" enctype="multipart/form-data">
-                <button type="submit" class="btn btn-info download" name="download" value="{{ @tutor['user_id'] }}">Downloads
+                <button type="submit" class="btn btn-info download" name="download" value="{{ @tutor['user_id'] }}">
+                    Download Image
                 </button>
+                <div class="mt-4">
```
Comment (dsloan92) [inline views/tutorInfo.html:21]: I would maybe add a line comment about what section of the page this div is.

File: model/Database.php [hunk 2]
```diff
@@ -1045,4 +1044,39 @@ function removeFile($itemId)
...
+                and Tutor.user_id = ?";
+        $statement = $this->_dbh->prepare($sql);
+        $statement->execute([$currentYear, $tutorId]);
+        return $statement->fetchAll(PDO::FETCH_ASSOC);
+    }
+
+    /**
+     * Get the yearId of the given tutor
+     * @param int $userId tutor's id
+     * @return mixed array of data in TutorYear and Tutor table
```
Comment (dsloan92) [inline model/Database.php]: if this function is to grab only the yearID of the given tutor, should we be returning more than the yearID?

**Review by dsloan92 — COMMENTED** [review]
Looks good Laxmi! Made two comments :)

File: views/tutorInfo.html (hunk 1 above)
Comment (laxmikandel97) [reply views/tutorInfo.html:21]: I will for sure! thank you

File: model/Database.php (hunk 2 above)
Comment (laxmikandel97) [reply model/Database.php]: No, but I was thinking if we need other stuff we don't have to write another function again. But we are almost done with the functionality so, I can just change that to just grab what I need. I will update it soon! Thanks

**Review by dsloan92 — APPROVED** [review]
Looks great! Thank you!! Looks like you uncommented the login bug I found too!

**Pull request merged**